scripts/cmake/CheckHeaderCompilation.cmake

```diff
@@ -1,6 +1,6 @@
 # Supply include directories and compiler flags
 get_directory_property(INCLUDE_DIRS INCLUDE_DIRECTORIES)
-set(CMAKE_REQUIRED_FLAGS "-c -std=gnu++14")
+set(CMAKE_REQUIRED_FLAGS "-c")
```

Comment:
I'm also not sure why we have this flag here, because `check_cxx_source_compiles` compiles and links the executable. `-c` would mean no executable is produced, only object file.
But if I take it out, I get a few new linker errors...

Reply:
Couldn't get it to work w/o '-c' flag... next time.